content/docs/compute/custom-image.md

```diff
@@ -0,0 +1,184 @@
+</Tabs>
+
+### Available parameters for creation
+
+| Parameter                 | Description                                                   |
+| ------------------------- | ------------------------------------------------------------- |
+| `--name`                  | Name of the disk image (required)                             |
+| `--distribution`          | OS distribution (e.g., "ubuntu", "centos") (required)         |
+| `--version`               | Version of the operating system (required)                    |
+| `--path`                  | Path to the disk image file (.raw or .qcow2) (required)       |
+| `--os` (optional)         | Operating system type (defaults to "linux", can be "windows") |
+| `--logo_path` (optional ) | Path to your logo file (SVG format)                           |
+
+## Checking disk image status
+
+<Tabs groupId="check-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+After uploading an image, it goes through several processing stages. You can check the status of your disk image with:
+
+```console
+$ civo diskimage show cust-my-ubuntu-image
+                ID : 8a76ec9c-ef94-4d21-90a5-b48fc6d4fee2
+              Name : cust-my-ubuntu-image
+      Distribution : ubuntu
+           Version : 22.04
+                OS : linux
+             State : available
+         Image URL : https://objectstore.staging.civo.com/disk-images/my-ubuntu-image/disk.raw?X-Amz-Algorithm=AWS4-HMAC-SHA256&SignedHeaders=host&x-id=PutObject&X-Amz-Signature=8a7d32f...
+          Logo URL : https://objectstore.staging.civo.com/disk-images/my-ubuntu-image/logo.svg
+       Image Size : 2.5 GB
+```
+
+</TabItem>
+</Tabs>
+
+### Understanding disk image states
+
+The disk image states include:
+
+- `upload_pending`: Initial state after creation, awaiting upload
+- `upload_completed`: Upload finished, MD5 checksum verified
+- `pending`: Image is being processed for distribution
+- `available`: Image is ready to use for creating instances
+- `upload_expired`: Upload not completed within 15 minutes
+- `error`: Issue with upload or processing
+
+:::note
+Only images in the **available** state can be used to create instances.
+:::
+
+## Listing custom disk images
+
+<Tabs groupId="list-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+To see all your custom disk images:
+
+```console
+$ civo diskimage list --local
++--------------------------------------+----------------------+--------------+---------+-----------+
+| ID                                   | Name                 | Distribution | Version | Status    |
++--------------------------------------+----------------------+--------------+---------+-----------+
+| 8a76ec9c-ef94-4d21-90a5-b48fc6d4fee2 | cust-my-ubuntu-image | ubuntu       | 22.04   | available |
++--------------------------------------+----------------------+--------------+---------+-----------+
+```
+
+The `--local` flag filters the list to show only your custom uploaded disk images, excluding system images.
```

Comment:
This is different than the `type=custom` on the API. Would be ideal to have the options be the same across the board.

Reply:
I don't really mind either way, but it shouldn't be `local` because that's not consistent with the API (and doesn't make sense to me either because why would it be "local"?)